java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_2/Robot.java

```diff
@@ -0,0 +1,174 @@
+package com.zoolatech.lecture2.tasks._2;
+
+public class Robot {
+    private final Room room;
+    Direction direction;
+    private int width;
+    private int height;
+
+    public Robot() {
+        this.room = new Room(1, 1);
```

Comment:
1. Try to use constructor overloading - it will simplify the code.
2. Room's width and height should be passed as constructor arguments

Reply:
Rewrite class to meet this conditions.